examples/todomvc/README.md

```diff
@@ -0,0 +1,18 @@
+# TODO MVC using wasm-bingen and web-sys
+
+[wasm-bindgen](https://github.com/rustwasm/wasm-bindgen) and [web-sys](https://rustwasm.github.io/wasm-bindgen/api/web_sys/) coded [TODO MVC](https://todomvc.com/)
+
+The code was rewritten from the [ECMA 6 version](http://todomvc.com/examples/vanilla-es6/).
+
+The core differences are:
```

Comment:
Thanks for writing this up! Could this be added as a page in the examples guide at `guide/src/examples`? The other examples round here have a bland README template as well for what to put in the repo here.